packages/react/playground/package.json

```diff
@@ -1,42 +1,23 @@
 {
-  "name": "@cloudinary/react-example",
-  "homepage": ".",
-  "version": "0.0.0",
+  "name": "playground",
```

Comment:
Should that be `react-playground` perhaps? I'm not sure if we will get into any package naming conflicts or not.